### DISCUSSION TITLE
如何在群晖NAS的Docker里使用到youtubedl 或者 ytdlp 音源？

### DISCUSSION THREAD

**Discussion by EzioAuditore2020** [1715173329/UnblockNeteaseMusic#810] [Q&A]
如何在群晖NAS的Docker里使用到youtubedl 或者 ytdlp 音源？ 

使用说明上说要自行安装 youtube-dl 或者 yt-dlp 请问在docker里如何安装这两个东西？

![Docker install yt-dlp](https://user-images.githubusercontent.com/60751422/182948975-5ce90232-0945-47b5-a52f-d2c7448c870d.png)

**Comment by EzioAuditore2020** [comment]
有人能回答一下这个问题吗？

**Reply by pan93412** [reply]
我沒有 NAS，不知道怎麼做…… 😞

試試看直接用我們提供的 Docker 映像？

**Reply by EzioAuditore2020** [reply]
看我截图，我已经在docker里安装你们提供的镜像了，但是不知道如何使用到youtube-dl或者yt-dlp音源，我等等看有没有会用群晖NAS的大神出现吧

**Reply by hncjygd** [reply]
威联通可以进入终端然后wget下载yt-dlp到docker即可。 群晖应该是一样的，我猜就是Terminal -> /bin/sh进去下载到/bin目录就好了。

**Reply by nyavana** [reply]
> 威联通可以进入终端然后wget下载yt-dlp到docker即可。 群晖应该是一样的，我猜就是Terminal -> /bin/sh进去下载到/bin目录就好了。

请问有文档/教程可以参考吗，我是用portainer部署，找了一圈没有查到怎么弄
